beacon_node/beacon_chain/src/beacon_chain.rs

```diff
@@ -4159,6 +4165,13 @@ impl<T: BeaconChainTypes> BeaconChain<T> {
         }
     }
 
+    /// Returns `true` if the given slot is prior to the `bellatrix_fork_epoch`.
+    fn slot_is_prior_to_bellatrix(&self, slot: Slot) -> bool {
```

Comment:
Should this logic just be "is bellatrix enabled". Because when we release the version of lighthouse that enables bellatrix I think we will want users seeing these `CRITs` as soon as possible if they don't have an EE.

Reply:
My plan is to create some other service that logs errors/crits based on a more fine-grained heuristic. E.g., start logging crits 2 weeks before the fork time.

I'm thinking that we might want to cut a release that includes the Bellatrix fork epoch without forcing everyone to update right then.

Reply:
SGTM, shall we merge this and release v2.2.1?

Reply:
Good with me!